var/spack/repos/builtin/packages/rocm-debug-agent/package.py

```diff
@@ -88,15 +90,20 @@ def cmake_args(self):
         spec = self.spec
         args = []
 
-        if '@3.5.0' in spec:
+        if self.spec.satisfies('@3.5.0'):
```

Comment:
use spec.satifies as spec=self.spec is already defined in line 90

Reply:
self.spec.satisfies changed to spec.satisfies in rocm-debug-agent recipe
line 93, 99, 104